web-server/web-server.markdown

```diff
@@ -1,7 +1,7 @@
 title: A Simple Web Server
 author: Greg Wilson
 
-_Greg Wilson is the founder of Software Carpentry, a crash course in computing skills for scientists and engineers.  He has worked for 30 years in in both industry and academia, and is the author or editor of several books on computing, including the 2008 Jolt Award winner "Beautiful Code" and the first two volumes of "The Architecture of Open Source Applications". Greg received a Ph.D. in Computer Science from the University of Edinburgh in 1993._
+_Greg Wilson is the founder of Software Carpentry, a crash course in computing skills for scientists and engineers.  He has worked for 30 years in both industry and academia, and is the author or editor of several books on computing, including the 2008 Jolt Award winner *Beautiful Code* and the first two volumes of *The Architecture of Open Source Applications*. Greg received a PhD in Computer Science from the University of Edinburgh in 1993._
```

Comment:
@MichaelDiBernardo If this bio is printed in italics, the titles of the books should be in roman, and vice versa.